specification/search/data-plane/Azure.Search/preview/2023-10-01-Preview/searchindex.json

```diff
@@ -0,0 +1,2815 @@
+          }
+        ]
+      },
+      "description": "This parameter is only valid if the query type is 'semantic'. If set, the query returns captions extracted from key passages in the highest ranked documents. When Captions is set to 'extractive', highlighting is enabled by default, and can be configured by appending the pipe character '|' followed by the 'highlight-<true/false>' option, such as 'extractive|highlight-true'. Defaults to 'None'."
+    },
+    "Vector": {
+      "type": "object",
+      "discriminator": "kind",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "The name of the kind of vector query being performed.",
+          "x-nullable": false
+        },
+        "k": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of nearest neighbors to return as top hits."
+        },
+        "fields": {
+          "type": "string",
+          "description": "Vector Fields of type Collection(Edm.Single) to be included in the vector searched."
+        },
+        "exhaustive": {
+          "type": "boolean",
+          "description": "If set to true, will compute the exact nearest neighbors by searching through all the training points returning the true k nearest neighbors."
+        }
+      },
+      "required": [
+        "kind"
+      ],
+      "description": "The query parameters for vector and hybrid search queries."
+    },
+    "RawVector": {
+      "type": "object",
+      "x-ms-discriminator-value": "vector",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Vector"
+        }
+      ],
+      "properties": {
+        "vector": {
+          "type": "array",
+          "items": {
+            "type": "number",
+            "format": "float"
+          },
+          "description": "The vector representation of a search query."
+        }
+      },
+      "description": "The query parameters to use for vector search when a raw vector value is provided."
+    },
+    "TextVector": {
+      "type": "object",
+      "x-ms-discriminator-value": "text",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Vector"
+        }
+      ],
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "The text to be vectorized to perform a vector search query."
+        }
+      },
+      "description": "The query parameters to use for vector search when a text value that needs to be vectorized is provided."
+    },
+    "VectorFilterMode": {
+      "type": "string",
+      "enum": [
+        "postFilter",
+        "preFilter"
+      ],
+      "x-ms-enum": {
+        "name": "VectorFilterMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "postFilter",
+            "name": "PostFilter",
+            "description": "The search data will be filtered after the vector search query is performed."
```

Comment:
```suggestion
            "description": "The filter will be applied after the candidate set of vector results is returned. Depending on the filter selectivity, this can result in fewer documents than requested by the parameter 'k'."
```